components/button/src/dropdown-button/dropdown-button.js

```diff
@@ -67,7 +68,18 @@ class DropdownButton extends Component {
     }
     anchorRef = React.createRef()
 
-    onToggle = ({ name, value }, event) => {
+    controlledOnClick = ({ name, value }, event) => {
```

Comment:
Having one handler that does both uncontrolled/controlled flows seems a bit simpler to me. :thinking: